geomstats/_backend/pytorch/__init__.py

```diff
@@ -155,13 +161,9 @@ def split(x, indices_or_sections, axis=0):
     return torch.split(x, tuple(intervals_length), dim=axis)
 
 
-def logical_or(x, y):
-    return x or y
-
-
 def logical_and(x, y):
```

Comment:
what is happening with codecov? the coverage is shown to decrease on this PR, and for example it seems that the __init__.py file has coverage 0%... :/